content/en/docs/concepts/configuration/resource-bin-packing.md

```diff
@@ -134,21 +134,21 @@ Node Score:
 intel.com/foo  = resourceScoringFunction((2+1),4)
                =  (100 - ((4-3)*100/4)
                =  (100 - 25)
-               =  75
-               =  rawScoringFunction(75)
-               = 7
+               =  75                      # requested + used = 75% * available
```

Comment:
The spacing I was talking about, some have space after "=" while some don't.

Reply:
Thanks @sudeshsh. I've fixed the spacing. Please take a quick look when you get a chance.